components/edit-collective/CreateHostForm.js

```diff
@@ -139,23 +139,22 @@ class CreateHostForm extends React.Component {
         {!host && (
           <Fragment>
             <CreateOrganizationForm onChange={org => this.handleChange('organization', org)} />
-            <Button
-              bsStyle="primary"
-              type="submit"
+            <StyledButton
+              buttonStyle="primary"
```

Comment:
We shoud keep this

```suggestion
              buttonStyle="primary"
              type="submit"
```

Reply:
@Betree : Done. Thanks for pointing it out. 👍🏽